src/documentation/reference/introduction/page-arguments.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Page Arguments
+order: 3
+---
+
+There are three arguments for customizing a page response.
+
+<Endpoint>
+
+|     |                                                                  |
+| --- | ---------------------------------------------------------------- |
+| GET | /{endpoint}?cursor={paging_token}&order={asc,desc}&limit={1-200} |
+
+</Endpoint>
+
+<NestedTable>
+  
+- cursor
+  - optional
+  - A number that points to a specific location in a collection of responses and is pulled from the `paging_token` value of a record.
+- order
+  - optional
+  - A designation of the order in which records should appear. Options include `asc`(ascending) or `desc` (descending). If this argument isn’t set, it defaults to `asc`.
+- limit
+  - optional
+  - The total number of records returned. The limit can range from 1 to 200 - an upper limit that is hardcoded in Horizon for performance reasons. If this argument isn’t designated, it defaults to 10.
+  
+</NestedTable>
+
+<RightTable title="Example Request" isCodeSnippet>
+  
+```cURL
+curl "https://horizon.stellar.org/ledgers/26478723/operations?cursor=113725249324879872&limit=5&order=asc"
+```
+
+</RightTable>
+
+<RightTable title="Example Response" isExampleResponse>
```

Comment:
What is `<RightTable/>`?

Reply:
typo. should be <RightColumn/>. should i fix now or wait until we update these component names?